ethereum-financial/src/clients/eth_provider.ts

```diff
@@ -53,12 +53,12 @@ export class ETHProvider implements IAaveClient, IPoolBalanceClient {
     }
   }
 
-  public async getTotalSupply(blockNumber: number): Promise<E.Either<Error, BigNumber>> {
+  public async getTotalSupply(blockHash: string): Promise<E.Either<Error, BigNumber>> {
     try {
       const out = await retryAsync<EtherBigNumber>(
         async (): Promise<EtherBigNumber> => {
           const [totalSupply] = await this.astEthContract.functions.totalSupply({
-            blockTag: blockNumber,
+            blockTag: blockHash,
```

Comment:
```suggestion
            blockHash: blockHash,
```

Sorry, but looks like the field name here should be `blockHash`: https://github.com/lidofinance/alerting-forta/blob/main/ethereum-steth/src/clients/eth_provider.ts#L181-L196